graylog2-server/src/main/java/org/graylog2/migrations/V20241212102900_ScopeMigration.java

```diff
@@ -44,11 +45,13 @@
 public class V20241212102900_ScopeMigration extends Migration {
     private static final Logger LOG = LoggerFactory.getLogger(V20241212102900_ScopeMigration.class);
 
+    private final ClusterConfigService configService;
     private final MongoCollection<EventDefinitionDto> eventDefinitionCollection;
     private final MongoCollection<IndexSetConfig> indexSetCollection;
 
     @Inject
-    public V20241212102900_ScopeMigration(MongoCollections mongoCollections) {
+    public V20241212102900_ScopeMigration(ClusterConfigService configService, MongoCollections mongoCollections) {
```

Comment:
One last thing: can we change the migration name, currently it dates back to last year

Reply:
Good point - talk about a long-running PR